{
  "NServiceBus.MessageId": "7280f019-e248-4443-8faf-a82b0090abe2",
  "NServiceBus.MessageIntent": "Send",
  "NServiceBus.ReplyToAddress": "ClientUI",
  "NServiceBus.CorrelationId": "7280f019-e248-4443-8faf-a82b0090abe2",
  "NServiceBus.OriginatingMachine": "DESKTOP-TQAN4CG",
  "NServiceBus.OriginatingEndpoint": "ClientUI",
  "$.diagnostics.originating.hostid": "f86ddf58f78971facf9f41e20417d4f0",
  "NServiceBus.ContentType": "text\/xml",
  "NServiceBus.EnclosedMessageTypes": "Messages.Commands.StartOrder, Messages, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
  "NServiceBus.ConversationId": "b7a615ba-19d0-435f-94db-a82b0090abe2",
  "NServiceBus.Version": "6.4.1",
  "NServiceBus.TimeSent": "2017-11-14 08:46:43:952555 Z",
  "NServiceBus.ExceptionInfo.ExceptionType": "System.InvalidOperationException",
  "NServiceBus.Retries.Timestamp": "2017-11-14 08:48:37:325018 Z",
  "NServiceBus.ExceptionInfo.HelpLink": null,
  "NServiceBus.ExceptionInfo.Message": "No handlers could be found for message type: Messages.Commands.StartOrder",
  "NServiceBus.ExceptionInfo.Source": "NServiceBus.Core",
  "NServiceBus.ExceptionInfo.StackTrace": "System.InvalidOperationException: No handlers could be found for message type: Messages.Commands.StartOrder\u000d\u000a   at NServiceBus.LoadHandlersConnector.<Invoke>d__1.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Pipeline\\Incoming\\LoadHandlersConnector.cs:line 31\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Pipeline\\Incoming\\DeserializeLogicalMessagesConnector.cs:line 33\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.ReceivePerformanceDiagnosticsBehavior.<Invoke>d__2.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Performance\\Statistics\\ReceivePerformanceDiagnosticsBehavior.cs:line 40\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Performance\\Statistics\\ProcessingStatisticsBehavior.cs:line 27\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Pipeline\\Incoming\\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 37\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Pipeline\\MainPipelineExecutor.cs:line 32\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\u000d\u000a   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Transports\\Learning\\LearningTransportMessagePump.cs:line 277",
  "NServiceBus.TimeOfFailure": "2017-11-14 08:49:08:648310 Z",
  "NServiceBus.FailedQ": "Sales",
  "NServiceBus.ProcessingMachine": "DESKTOP-TQAN4CG",
  "NServiceBus.ProcessingEndpoint": "Sales",
  "$.diagnostics.hostid": "f0537806032cb850d763396e2821a608",
  "$.diagnostics.hostdisplayname": "DESKTOP-TQAN4CG"
}